packages/jasmine-enzyme/README.md

```diff
@@ -41,36 +42,36 @@ describe('test', () => {
 
 ### Assertions
 
-> * Not all assertions work with every rendering strategy.
+> - Not all assertions work with every rendering strategy.
 >   If you are wondering what rendering mechanism to use when, refer to
 >   [enzyme's documentation](https://github.com/airbnb/enzyme).
 
-* [toBeChecked()](#tobechecked)
-* [toBeDisabled()](#tobedisabled)
-* [toBeEmptyRender()](#tobeemptyrender)
-* [toExist()](#toexist)
-* [toContainMatchingElement()](#tocontainmatchingelement)
-* [toContainMatchingElements()](#tocontainmatchingelements)
-* [toContainExactlyOneMatchingElement()](#tocontainexactlyonematchingelement)
-* [toContainReact()](#tocontainreact)
-* [toHaveClassName()](#tohaveclassname)
-* [toHaveDisplayName()](#tohavedisplayname)
-* [toHaveHTML()](#tohavehtml)
-* [toHaveProp()](#tohaveprop)
-* [toHaveRef()](#tohaveref)
-* [toHaveState()](#tohavestate)
-* [toHaveStyle()](#tohavestyle)
-* [toHaveTagName()](#tohavetagname)
-* [toHaveText()](#tohavetext)
-* [toIncludeText()](#toincludetext)
-* [toHaveValue()](#tohavevalue)
-* [toMatchElement()](#tomatchelement)
-* [toMatchSelector()](#tomatchselector)
+- [toBeChecked()](#tobechecked)
```

Comment:
seems like a lot of unrelated changes?

Reply:
Thank you! Reverted and disabled prettier